documentation/source/newsfragments/1424.feature.rst

```diff
@@ -0,0 +1 @@
+Questa now supports :envvar:`PLUSARGS`. This requires that ``tcl.h`` be present on the system. This is likely included in your installation of Questa. Otherwise, specify ``CXXFLAGS=-I/path/to/tcl/includedir``.
```

Comment:
Does this `CXXFLAGS` thing still work with the setup.py changes?

Reply:
Yes. All should work.

Reply:
This works so should change `CFLAGS="-I/path_to_mti_h" pip install -v -e .`

Reply:
The file that includes tcl.h is a C++ file, it's curious that `CFLAGS` works, or is this due to setuptools not supporting `CXXFLAGS` and only using `CFLAGS`?

Reply:
setuptools seems to support CFLAGS and not CXXFLAG

Reply:
Alright, will change.